packages/react-jsx-highcharts/src/utils/getBoundChartMethod.js

```diff
@@ -0,0 +1,11 @@
+export default function getBoundChartMethod(chart, method, context) {
+  const boundMethod = method.bind(context);
+
+  return function (...rest) {
+    if (!chart.__destroyed) {
+      return boundMethod(...rest)
+    }
+
+    return null;
```

Comment:
P.s. I wonder if we returned a noop function (`noop = () => {}`) here instead, we might not need to

```js
componentWillUnmount () {
  const { remove } = this.props;

  remove && remove();
}
```

We could just call `this.props.remove()` as before - it just wouldn't do anything.